apps/yarn.lock

```diff
@@ -60,7 +60,7 @@
     temporal latest
   optionalDependencies:
     browser-serialport latest
-    firmata "^0.12.0"
+    firmata latest
```

Comment:
I misspoke: I don't think we have to require `firmata` ourselves to have them resolve to the same version, npm/yarn is smart enough to do that for us now that the versions overlap. The reason for the direct dependency is that our code imports it.